ci/scripts/integration_arrow.sh

```diff
@@ -63,10 +63,15 @@ time archery integration \
     --run-ipc \
     --run-flight \
     --with-cpp=$([ "$ARROW_INTEGRATION_CPP" == "ON" ] && echo "1" || echo "0") \
+    --target-language=cpp \
     --with-csharp=$([ "$ARROW_INTEGRATION_CSHARP" == "ON" ] && echo "1" || echo "0") \
+    --target-language=csharp \
     --with-go=$([ "$ARROW_INTEGRATION_GO" == "ON" ] && echo "1" || echo "0") \
+    --target-language=go \
     --with-java=$([ "$ARROW_INTEGRATION_JAVA" == "ON" ] && echo "1" || echo "0") \
+    --target-language=java \
     --with-js=$([ "$ARROW_INTEGRATION_JS" == "ON" ] && echo "1" || echo "0") \
+    --target-language=js \
```

Comment:
This file is used by apache/arrow-rs too.
So embedding `--target-language` here isn't a good approach.